### PR TITLE
[Oy2-9914] Create APM-CH Measure

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Create the measure APM-CH.

#### Linked Issues to Close

[Oy2-9914](https://qmacbis.atlassian.net/browse/OY2-9914)

#### Pull Request Creator Checklist

- [X] This PR has an associated issue or issues.
- [X] The associated issue(s) are linked above.
- [X] This PR meets all acceptance criteria for those issues.
- [X] This PR and linked issue(s) are adequately documented
- [X] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [ ] Someone has been assigned this PR.
- [ ] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/APMCH/validation.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+const APMCHValidation = (data: FormData) => {
+  const dateRange = data[DC.DATE_RANGE];
+  const deviationArray = GV.getDeviationNDRArray(
+    data.DeviationOptions,
+    data.Deviations,
+    true
+  );
+  const didCalculationsDeviate = data[DC.DID_CALCS_DEVIATE] === DC.YES;
+  const includesHybridDataSource = data[DC.DATA_SOURCE]?.includes(
+    DC.HYBRID_ADMINSTRATIVE_AND_MEDICAL_RECORDS_DATA
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/APMCH/validation.ts:16]: Don't need Hybrid Data Source check for non-hybrid measures

File: services/ui-src/src/measures/2021/APMCH/validation.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    ...GV.validateNumeratorsLessThanDenominators(
+      performanceMeasureArray,
+      OPM,
+      PMD.qualifiers
+    ),
+    ...GV.validateNoNonZeroNumOrDenom(
+      performanceMeasureArray,
+      OPM,
+      PMD.qualifiers,
+      includesHybridDataSource
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/APMCH/validation.ts:40]: unnecessary based on above

File: services/ui-src/src/measures/2021/APMCH/validation.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        OMSData(true),
+        PMD.qualifiers,
+        PMD.categories
+      ),
+      validationCallbacks: [
+        GV.validateDenominatorGreaterThanNumerator,
+        GV.validateDenominatorsAreTheSame,
+        GV.validateRateNotZero,
+        GV.validateOMSTotalNDR,
+        ...(includesHybridDataSource ? [] : [GV.validateRateZero]),
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/APMCH/validation.ts:68]: unnecessary based on above

File: services/ui-src/src/measures/2021/APMCH/validation.ts (hunk 1 above)
Comment (jphechter) [reply services/ui-src/src/measures/2021/APMCH/validation.ts:16]: included

File: services/ui-src/src/measures/2021/APMCH/validation.ts (hunk 3 above)
Comment (jphechter) [reply services/ui-src/src/measures/2021/APMCH/validation.ts:68]: included

**Pull request merged**